docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -0,0 +1,213 @@
+```sql
+ALTER ROLE db_datareader ADD MEMBER [sqlserver-identity];
+```
+
+<Admonition type="note">
+Since we’re creating contained users, this step needs to be done on each
+database we want to be able to log in.
+</Admonition>
+
+## Step 6/7. Start Teleport Database Service
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=teleport.example.com:3080 \
+   --name=sqlserver \
+   --protocol=sqlserver \
+   --uri=my-server.database.windows.net:1433 \
+   --labels=env=dev
+```
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=mytenant.teleport.sh:443 \
+   --name=sqlserver \
+   --protocol=sqlserver \
+   --uri=my-server.database.windows.net:1433 \
+   --labels=env=dev
+```
+</ScopedBlock>
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple different services, for example
+  multiple Database Service agents as well as the SSH Service or Application
+  Service.
+  
+</Admonition>
+
+Start the database service:
+
+```code
+$ teleport start --config=/etc/teleport.yaml
+```
+
+## Step 7/7. Connect
+
+Login into your Teleport Cluster. Your database should appear in the list of
+available databases:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name      Description         Labels
+# --------- ------------------- -------
+# sqlserver                     env=dev
+```
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+$ tsh db ls
+# Name      Description         Labels
+# --------- ------------------- -------
+# sqlserver                     env=dev
+```
+</ScopedBlock>
+
+To retrieve credentials for a database and connect to it:
+
+```code
+$ tsh db connect --db-user=sqlserver-identity --db-name=master sqlserver
+```
+
+Where `--db-user` is the managed identity name.
+
+<Admonition type="note">
+  The `mssql-cli` command-line client should be available in `PATH` of the machine
+  you're running `tsh db connect` from.
+</Admonition>
+
+## Troubleshooting
+
+### Could not find identity
+
+If you see the error `could not find identity "my-identity" attached to the
+instance` when connecting to your database, then the identity you’re trying to
+connect with is not attached to the Teleport Database Service Virtual Machine.
+You can navigate to the [Virtual Machines](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines),
+open the **Teleport instance**, **Identity** section, and choose **User
+assigned** to see all identities you can connect with. If you don’t see your
+identity check [Step 4](#step-47-configure-azure-managed-identities) to see how
```

Comment:
If following Step 4 correctly should prevent this error, do we need to include this troubleshooting step? Are there any instructions we could add to Step 4 to make it less likely for someone to hit this error?

Reply:
> Are there any instructions we could add to Step 4 to make it less likely for someone to hit this error?

I think Step 4 has all the information necessary. I've added this section for "returning users" where they try to connect to a non-configured user. I wasn't sure if this guide was the right place. Maybe place it on the [database troubleshooting page](https://goteleport.com/docs/database-access/troubleshooting/)? What do you think?

Reply:
That makes sense—I think we can keep this as it is!